sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala

```diff
@@ -288,4 +288,33 @@ class HiveOrcQuerySuite extends OrcQueryTest with TestHiveSingleton {
       }
     }
   }
+
+  test("SPARK-32234: orc data created by the hive tables having _col fields name" +
```

Comment:
we can shorten the test name: `SPARK-32234: read ORC table with column names all starting with '_col'`

Reply:
done